internal/webhook/external/github.go

```diff
@@ -164,7 +165,18 @@ func (g *githubWebhookReceiver) getHandler(requestBody []byte) http.HandlerFunc
 				xhttp.WriteErrorJSON(w, err)
 				return
 			}
-			repoURL = image.NormalizeURL(ref.Context().Name())
+			repoURL := image.NormalizeURL(ref.Context().Name())
+			repoURLs = append(
+				repoURLs,
+				repoURL,
+				// This covers our bases in the event that the package event originated
+				// from an OCI repository that actually contains Helm charts. All OCI
+				// Helm chart repositories in Kargo start with oci://. So we're going to
+				// try to refresh Warehouses subscribed to both, e.g.,
+				// ghcr.io/example/repo AND oci://ghcr.io/example/repo. Only one of the
+				// two should actually work.
+				helm.NormalizeChartRepositoryURL(fmt.Sprintf("oci://%s", repoURL)),
```

Comment:
Shouldn't this be:


```suggestion
				fmt.Sprintf("oci://%s", helm.NormalizeChartRepositoryURL(repoURL)),
```

As `NormalizeChartRepositoryURL` strips the `oci://` prefix, and adding it seems otherwise redundant.

Reply:
Disregard. I hate how I did this whole thing.